odoo/custom/src/repos.yaml

```diff
@@ -53,6 +53,19 @@ openspp_registry:
     - openspp 15.1.1
     # Example of a merge of the PR with the number <PR>
     # - oca refs/pull/<PR>/head
+openspp_api:
+  defaults:
+    # Shallow repositories ($DEPTH_DEFAULT=1) are faster & thinner
+    # You may need a bigger depth when merging PRs (use $DEPTH_MERGE
+    # for a sane value of 100 commits)
+    depth: $DEPTH_DEFAULT
+  remotes:
+    openspp: https://github.com/openspp/openspp-api.git
+  target: openspp 15.1.1
+  merges:
+    - openspp spp_dci_api_server
```

Comment:
Should we do `15.1.1` instead

Reply:
Valid point. I updated this.

Reply:
However, there is no `15.1.1` in openspp-api – only `15.0`. How do you suggest that we proceed @jeremi ?